src/client/dfuse/SConscript

```diff
@@ -32,7 +32,7 @@ OPS_SRC = ['create',
            'write',
            'statfs']
 
-IOIL_SRC = ['int_posix.c', 'int_read.c', 'int_write.c']
+IOIL_SRC = ['int_posix.c', 'int_read.c', 'int_write.c', '../pil4dfs/hook.c']
```

Comment:
I'm surprised you don't get build warnings with this approach.   Typically, one would do something like build pil4dfs first and 

interception_targets = env.SharedObject('hook.c')
Export('interception_targets')

And in ioil SConscript
Import('interception_targets')


Then use interception_targets in source list of both libraries.

Or something like that.

Reply:
Maybe better build hook.c here and then put the variable in both lists

Reply:
Thank you very much! Just updated as your suggested!